download.html

```diff
@@ -23,10 +23,12 @@
   <p>TripleA for Linux</p>
 </a>
 
-<p>Upgrading from 1.8.0.9? <a href="{{ '/upgrade_notes/' | prepend: site.baseurl }}">read here for upgrade notes</a></p>
-<p>See what has changed in the <a href="{{ '/release_notes/' | prepend: site.baseurl }}">release notes</a></p>
-<p>Download <a href="{{ '/old_downloads/' | prepend: site.baseurl }}">previous TripleA releases here</a></p>
-<p>Download <a id="sourceLink" href="https://github.com/triplea-game/triplea/releases/latest">Source Code from Github Releases</a></p>
+<h2>Additional Links</h2>
+<p><a href="{{ '/release_notes/' | prepend: site.baseurl }}">Release Notes</a></p>
+<p><a id="sourceLink" href="https://github.com/triplea-game/triplea/releases/latest">Source Code</a></p>
+<p><a href="{{ '/old_downloads/' | prepend: site.baseurl }}">Previous TripleA Releases</a></p>
+<p><a id="sourceLink" href="https://github.com/triplea-game/triplea/releases/">Early Release Download</a></p>
```

Comment:
You used some duplicate ids here which caused the travis build to fail.
You should probably use classes instead